src/pages/dashboard/_CDCComparisonContainer.js

```diff
@@ -66,8 +66,13 @@ export default function CDCComparisonContainer() {
           sample.
         </p>
       </div>
+      <h3>DAILY NEW TESTS IN THE U.S.</h3>
```

Comment:
Let's make this normal text case and use CSS to transform it. Also, I know its weird but we're using "US" instead of "U.S." for consistency

Reply:
This is already added upstream with more recent styling.

https://github.com/COVID19Tracking/website/blob/a3c01a98245792e168daffc4c5ded5ac38b554a0/src/pages/dashboard/_CDCComparisonContainer.js#L129-L131